src/js/play.js

```diff
@@ -111,22 +105,22 @@ var playState = {
         if(egg.key === "egg"){
             gameController.tweenEgg("crackedEgg", egg);
             gameController.eggCrack.play();
-            this.updateScoreAndPlayAnimation(-gameController.regularEggPoints);
+            this.updateScoreAndPlayAnimation(0); //when eggs are missed, nothing happens to score
```

Comment:
Wouldn't it be simpler to just remove these score update calls? That would also mean there's no need to check if `points > 0` in `updateScoreAndPlayAnimation`